documentation/design/security-metadata.md

```diff
@@ -215,7 +227,7 @@ First two presented option are not to be used. All the other options will likely
 </Target>
 ```
 
-Should we redact all occurences of value of `MySecret` from the task result? We might get a lot of false positives and very confusing results.
+In case we'd want to redact all occurences of value of `MySecret` from the task result - we might get a lot of false positives and very confusing results.
```

Comment:
Good point here.

We might end up not exposing the regex and plugin model for external usage; or we may open it with disclaimer of the perf implications and log messing are fully responsibility of users.

Primary mode of operation should be explicit indication of properties/items which are meant to hold secrets. Plus the built-in high-accuracy detection logic that will mainly serve for the feature discoverability by users ("oh I got something redacted in my log - let me search why - oh, there is this feature and I can denote my secrets holders and then turn off the patterns scanning") and bridging for the time until the feature is sufficiently addopted (this may very likely mean it will stick forever :-))